ci-operator/step-registry/openshift-qe/installer/gcp/openshift-qe-installer-gcp-workflow.yaml

```diff
@@ -0,0 +1,10 @@
+workflow:
+  as: openshift-qe-installer-gcp
+  steps:
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
```

Comment:
I think the name 'openshift-qe-installer-gcp' make people who wants to reuse it thinks this is just for install. So 'create-infra-move-ingress-monitoring-registry' is not expected here. I suggest to put 'create-infra-move-ingress-monitoring-registry' in perfscale's test config.

Reply:
+1

Reply:
Hi @qiliRedHat @liqcui  👋 ,
In a "self-managed" OCP, the infra components do not have designated nodes,
this chain simplifies that `create-infra-move-ingress-monitoring-registry`, in order to resonate a managed-service/ROSA like deployment, hence `create-infra-move-ingress-monitoring-registry` was captured into
the same workflow, which is originally inspired from https://github.com/openshift/release/pull/43601